fh-sync.d.ts

```diff
@@ -0,0 +1,407 @@
+// Type definitions for fh-sync
+// Project: https://github.com/feedhenry/fh-sync
+// Maintainer feedhenry-dev@redhat.com
+
+declare module SyncCloud {
```

Comment:
I'm going to check if all dependent projects compile after this change. 
We may also bump minor version.